tests/Basic/Basic.h

```diff
@@ -760,15 +760,10 @@ class TestNamingAnonymousTypesInUnion
 };
 
 #define CS_API
-class CS_API ClassPassTry
-{
-public:
-	int n;
-	char c;
-};
+class CS_API RefTypeClassPassTry { };
```

Comment:
Do you remember what the issue with `CS_API` was? At first I was thought that it was being specially processed by `CheckMacrosPass` or something but I checked the source that seemed not to be the case.